packages/embark/src/lib/modules/ethereum-blockchain-client/index.js

```diff
@@ -184,6 +185,32 @@ class EthereumBlockchainClient {
     });
   }
 
+  async determineAccounts(params, callback) {
+    let provider = await this.events.request2("blockchain:client:provider", "ethereum");
+    let web3 = new Web3(provider)
+    let accounts = await web3.eth.getAccounts();
+    let deploymentAccount = accounts[0];
+    let contract = params.contract;
+
+    // applying deployer account configuration, if any
+    if (typeof contract.fromIndex === 'number') {
+      deploymentAccount = accounts[contract.fromIndex];
+      if (deploymentAccount === undefined) {
+        return callback(__("error deploying") + " " + contract.className + ": " + __("no account found at index") + " " + contract.fromIndex + __(" check the config"));
+      }
+    }
+    if (typeof contract.from === 'string' && typeof contract.fromIndex !== 'undefined') {
+      self.logger.warn(__('Both "from" and "fromIndex" are defined for contract') + ' "' + contract.className + '". ' + __('Using "from" as deployer account.'));
+    }
+    if (typeof contract.from === 'string') {
+      deploymentAccount = contract.from;
+    }
+
+    deploymentAccount = deploymentAccount || accounts[0];
+    contract.deploymentAccount = deploymentAccount;
```

Comment:
```suggestion
    contract.deploymentAccount = deploymentAccount || accounts[0];;
```
Can get rid of line 209